tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -723,3 +744,24 @@ func findInstalledCSV(expectedCSV string) bool {
 
 	return false
 }
+
+// ClusterVersionXYZ returns the cluster's OCP version.
+func ClusterVersionXYZ(clusterObj cluster.APIClientGetter) (string, error) {
```

Comment:
It seems like this function isn't really used. It's only passed as an argument to ClusterVersionXY. Why not just add this logic to ClusterVersionXY?

Reply:
because I use the ClusterVersionXY twice.
Once I pass to it the returned value from ClusterVersionXYZ
Another time the version from ibu

Reply:
rewrote - this function is gone.